plasmapy/particles/particle_class.py

```diff
@@ -368,7 +333,13 @@ def __init__(
             if self.element and self._attributes['integer charge']:
                 categories.add('ion')
 
-            # Element properties
+##########################
+
+def __setElementProperties__( self,
```

Comment:
[PEP8: for methods we use lowercase with underscore spacing](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)

This method also does not reach the standard for a dunder, so a single leading underscore will suffice, i.e. ``__setElementProperties__ -> _set_element_properties``